ios/Podfile.lock

```diff
@@ -13,4 +13,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 63cb0d0aef536c8cbf15bf664fcfe31852a13dd1
 
-COCOAPODS: 1.5.3
+COCOAPODS: 1.6.1
```

Comment:
please add this to the documentation for mobile dev environment

Reply:
https://github.com/mattermost/mattermost-developer-documentation/pull/296